src/main/java/org/owasp/esapi/User.java

```diff
@@ -380,7 +380,7 @@ public interface User extends Principal, Serializable {
     /**
      * Set the time of the last failed login for this user.
      *
-     * @param lastFailedLoginTime the date and time when the user just failed to login correctly.
+     * @param lastFailedLoginTime the date and time when the user just failed to log in correctly.
```

Comment:
No. Let's leave this as 'login'. The word login is even in the Cambridge dictionary and if people are searching, they likely would search for 'login' rather than 'log in'.

Reply:
Hey Kevin. https://dictionary.cambridge.org/dictionary/english-german/log-in?q=login
as far as I see it, cambridge also lists it as “log in” rather than “login”.
“login” is a noun, and “log in” is a verb.
https://www.reddit.com/r/webdev/comments/aml4hb/login_vs_log_in_are_we_using_these_wrong/
https://www.grammar.com/log_in_vs._login

Of course I can revert this commit, if you please, but I am still convinced that “log in” looks and sounds better when reading the sentence.

Reply:
How about we change it to "authenticate" then? Part of my reason is if people are searching through Javadoc for a term, I think they are unlikely to use "log in" or "sign on" because they are used to seeing them in login forms as a single word.